src/main/kotlin/OSSLLMCompletionProvider.kt

```diff
@@ -13,16 +13,21 @@ class OSSLLMCompletionProvider : InlineCompletionProvider {
 
     override suspend fun getProposals(request: InlineCompletionRequest): List<InlineCompletionElement> {
         try {
-            val proposal = Requester.getModelSuggestions(request.document.text).text ?: return emptyList()
+            val caretPosition = runReadAction {
+                request.editor.caretModel.primaryCaret.selectionStart
```

Comment:
I wonder why is this used and not, say `request.startOffset + 1` - what is the difference?